mods/taskbar-idle-clean-desktop.wh.cpp

```diff
@@ -0,0 +1,127 @@
+// ==WindhawkMod==
+// @id              taskbar-idle-clean-desktop
+// @name            Clean Desktop on Idle
+// @description     Hides the taskbar and minimizes all windows after a period of user inactivity. Includes smart detection.
+// @version         1.7
+// @author          Breiq
+// @include         explorer.exe
```

Comment:
See: https://github.com/ramensoftware/windhawk/wiki/Mods-as-tools:-Running-mods-in-a-dedicated-process